From f7717b7fe15a6bec5dadc60e95522e0a3c4d5bd3 Mon Sep 17 00:00:00 2001
From: Sinthu Raja <sinthu.raja@ti.com>
Date: Wed, 20 Dec 2023 10:39:41 +0530
Subject: [PATCH 2/2] arm: configs: omap5: Enable support for DCAN devices

Enable configs CAN_C_CAN and CAN_C_CAN_PLATFORM as module to add support
for C_CAN/D_CAN devices connected to the platform bus

Signed-off-by: Sinthu Raja <sinthu.raja@ti.com>
---
 arch/arm/configs/multi_v7_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/configs/multi_v7_defconfig b/arch/arm/configs/multi_v7_defconfig
index 34787d58870d..6250f587e081 100644
--- a/arch/arm/configs/multi_v7_defconfig
+++ b/arch/arm/configs/multi_v7_defconfig
@@ -1308,3 +1308,5 @@ CONFIG_PRINTK_TIME=y
 CONFIG_MAGIC_SYSRQ=y
 CONFIG_DEBUG_FS=y
 CONFIG_SPI_TI_QSPI=y
+CONFIG_CAN_C_CAN=m
+CONFIG_CAN_C_CAN_PLATFORM=m
-- 
2.36.1

